provider/image.go

```diff
@@ -680,8 +688,10 @@ func marshalArgs(a resource.PropertyValue) map[string]*string {
 	if !a.IsNull() {
 		for k, v := range a.ObjectValue() {
 			key := fmt.Sprintf("%v", k)
-			vStr := v.StringValue()
-			args[key] = &vStr
+			if !v.ContainsUnknowns() {
+				vStr := v.StringValue()
+				args[key] = &vStr
+			}
```

Comment:
Nit: keeping things left indented
```suggestion
			if v.ContainsUnknowns() {
				continue
			}
			vStr := v.StringValue()
			args[key] = &vStr

```